### ISSUE TITLE
[Feature]: Implement REST API Backend with MongoDB Integration

### ISSUE THREAD

**Issue by VinayakVispute** [sourabhsikarwar/Scene-Movie-Platform#135]
- [ X ] Would you like to take up this task? 
- [ X ] Are you from GSSoC'23? 

|   ✨  |   Feature request                |
|-----|---------------------------------|
| **Is your feature request related to a problem?** | The website lacks a robust backend system that can handle data retrieval from MongoDB and expose it through a REST API and Show on the Website according to different category |
| **Describe the solution you'd like** | The solution involves designing and implementing a REST API backend that connects to a MongoDB database. The backend should be capable of performing CRUD (Create, Read, Update, Delete) operations on the data stored in MongoDB and exposing it through well-defined API endpoints and Uploading data to a efficient form <br><strong>Here are the key components of the solution: </strong> <br> 1. Database Schema Design <br> 2. API Endpoint Design <br> 3. Data Retrieval <br> 4. Data Manipulation |
| **Describe alternatives you've considered** |  Evaluating alternative databases or utilizing existing backend frameworks with MongoDB integration and REST API development capabilities |
| **Additional context** |    |

**Comment by geekaryan** [comment]
Yes I have prior knowledge of the API Integration and making backend of the applications.
Yes I am a contributor in GSSOC.
I am going to implement it using REST Framework, taking mongoDB as our database.
Assign this issue to me.
Thank you
